maps/southern_cross/southern_cross-4.dmm

```diff
@@ -439,7 +439,7 @@
 "iz" = (/obj/effect/floor_decal/industrial/warning/dust,/obj/structure/ore_box,/turf/simulated/floor/tiled/asteroid_steel,/area/surface/outpost/mining_main/cave)
 "iA" = (/obj/effect/step_trigger/teleporter/mine/from_mining,/turf/simulated/mineral/floor/ignore_mapgen/sif,/area/surface/cave/explored/normal)
 "iB" = (/obj/effect/step_trigger/teleporter/mine/from_mining,/turf/simulated/floor/water{outdoors = 0},/area/surface/cave/explored/normal)
-
+
```

Comment:
Miiiiiiight want to remove any whitespaces here as it may cause a whole row of space turfs. Same for below at the end of the file.

Reply:
I'm not sure why the mapmerger did this on the map files, but it looks like it just replaced a blank line with the same blank line?